src/library/autodj/autodjprocessor.cpp

```diff
@@ -1417,6 +1445,10 @@ void AutoDJProcessor::playerTrackLoaded(DeckAttributes* pDeck, TrackPointer pTra
                     // which may calls the calculateTransition() again without seek = true;
                     pDeck->setPlayPosition(pDeck->startPos);
                 }
+                if (!fromDeck->isPlaying() && fromDeck->playPosition() >= 1.0) {
```

Comment:
> 1.0?

Reply:
Yes, here we are in the relative domain.

Reply:
GitHub markdown: **\>** (greater than) or **\>=** (greater or equal)?

Reply:
greater or equal is correct here, because the playposition might be a fraction of one callback buffer ahead of 1.0 . 
I will drop a comment.